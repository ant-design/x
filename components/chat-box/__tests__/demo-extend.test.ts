import { extendTest } from '../../../tests/shared/demoTest';

extendTest('chat-box');
