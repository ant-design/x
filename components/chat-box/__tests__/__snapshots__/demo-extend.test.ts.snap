// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders components/chat-box/demo/basic.tsx extend context correctly 1`] = `
<div>
  2333
</div>
`;

exports[`renders components/chat-box/demo/basic.tsx extend context correctly 2`] = `
[
  "Warning: forwardRef render functions accept exactly two parameters: props and ref. %s",
]
`;
